fvm/fvm_test.go

```diff
@@ -1558,6 +1558,12 @@ func TestSignatureVerification(t *testing.T) {
 				return hash.NewSHA2_256()
 			},
 		},
+		{
+			"KECCAK_256",
+			func() hash.Hasher {
+				return hash.NewKeccak_256()
+			},
```

Comment:
This could be written as:
```suggestion
			"KECCAK_256",
			hash.NewKeccak_256,
```
and same for lines 1551 and 1557 (even though they are not part of this PR)

Reply:
updated